simpleheat.js

```diff
@@ -53,7 +53,7 @@ simpleheat.prototype = {
     },
 
     radius: function (r, blur) {
-        blur = blur || 15;
+        blur = blur === undefined ? 15 || blur;
```

Comment:
Oops, that doesn't look like a correct syntax. There should be `:` in place of `||` for proper ternary operator.